### PR TITLE
RULEAPI-717: Make sure all rules known to be duplicates on Jira are marked as such on github

### PR DESCRIPTION
**Pull request by arseniy-sonar**
S1219 - the duplication link is inverted (it is S915 that is the duplicate as it has the "Duplicate" Resolution)
S128 - the duplication link is inverted (it is S915 that is the duplicate as it has the "Duplicate" Resolution)
S131 - the duplication link is inverted (it is S915 that is the duplicate as it has the "Duplicate" Resolution)
S1684 - is a true duplicate of S134, but linked as "relates to" instead of "duplicates"
S2193 - the duplication link is inverted (it is S887 that is the duplicate as it has the "Duplicate" Resolution)
S3275 - the duplication link (with S3329) is inverted (marked as "is duplicated by", should be "duplicates")
S3430 - is a true duplicate of S3433, but not linked explicitly (only in a comment)
S3502 - is a true duplicate of S4144, but not linked explicitly (only in a comment)
S4880 - is a true duplicate of S1279, but not linked explicitly (only in a comment)
S5403 - is a true duplicate of S5018, but not linked explicitly (only in a comment)
S5424 - is a true duplicate of S3230, but not linked explicitly (only in a comment)
S5957 - is a true duplicate of S6007, but not linked explicitly (only in a comment)
S938 - is a true duplicate of S942, but that not linked (only in a comment)
S6175 - is a true duplicate of S878, but linked only with "relates to"
S869 - is a true duplicate of S5302, but linked as "relates to". add the "duplicates" link on github
S896 - is a true duplicate, but not closed for some reason. close it on github
S1480 - is a true duplicate, but has the "is replaced by" link to S1573 instead of "duplicates"
S858 - is a true duplicate of S1944, but linked as "is replaced by". add S1944 to the "replacementRules" in S858 metadata
S870 - is a true duplicate of S1944, but linked as "is replaced by". add S1944 to the "replacementRules" in S870 metadata
S913 - is a true duplicate, but linked as "is replaced by". add S121 to the "replacementRules" in S913 metadata
S887 - the duplication link is inverted (it is S887 that is the duplicate as it has the "Duplicate" Resolution)
S915 - the duplication link is inverted (it is S915 that is the duplicate as it has the "Duplicate" Resolution)

### CODE REVIEW

File: rules/S2161/java/metadata.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   ],
   "extra": {
     "replacementRules": [
-
+      "RSPEC-2162"
```
Comment (marco-antognini-sonarsource) [inline rules/S2161/java/metadata.json:14]: ditto, `tags` isn't empty.

File: rules/S2220/metadata.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   ],
   "extra": {
     "replacementRules": [
-
+      "RSPEC-2259"
```
Comment (marco-antognini-sonarsource) [inline rules/S2220/metadata.json:15]: ditto, `tags` isn't empty.

File: rules/S2700/metadata.json [hunk 3]
```diff
@@ -11,7 +11,7 @@
   ],
   "extra": {
     "replacementRules": [
-
+      "RSPEC-2187"
```
Comment (marco-antognini-sonarsource) [inline rules/S2700/metadata.json:14]: ditto, `tags` isn't empty.

File: rules/S913/metadata.json [hunk 4]
```diff
@@ -7,7 +7,7 @@
   ],
   "extra": {
     "replacementRules": [
-
+      "RSPEC-121"
```
Comment (marco-antognini-sonarsource) [inline rules/S913/metadata.json:10]: ditto, `tags` isn't empty.

**Review by marco-antognini-sonarsource — APPROVED** [review]
LGTM, and kudos for the energy you put into this PR: I haven't found a mistake!

I have 6 instances of the same question. Either way, the fix (do nothing or remove a few lines) is trivial so no need to do another round. I'm still interested to learn the answer to the question though. :-)

File: rules/S1480/plsql/metadata.json [hunk 5]
```diff
@@ -12,7 +12,7 @@
   ],
   "extra": {
     "replacementRules": [
```
[the remark this answers is not in the document]
Comment (arseniy-sonar) [reply rules/S1480/plsql/metadata.json:14]: There are many rspecs violating this policy.
I think we need to review it and either relax it, or enforce it in the CI.
I created a ticket for that: RULEAPI-719

[the remark this answers is not in the document]
Comment (marco-antognini-sonarsource) [reply rules/S1480/plsql/metadata.json:14]: Makes sense to me. 👍

**Pull request merged**